packages/loader/container-loader/src/container.ts

```diff
@@ -1695,31 +1695,36 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     private submitContainerMessage(type: MessageType, contents: any, batch?: boolean, metadata?: any): number {
-        const outboundMessageType: string = type;
-        switch (outboundMessageType) {
+        switch (type) {
             case MessageType.Operation:
-            case MessageType.RemoteHelp:
-                break;
-            case MessageType.Summarize: {
-                // github #6451: this is only needed for staging so the server
-                // know when the protocol tree is included
-                // this can be removed once all clients send
-                // protocol tree by default
-                const summary = contents as ISummaryContent;
-                if (summary.details === undefined) {
-                    summary.details = {};
-                }
-                summary.details.includesProtocolTree =
-                    this.options.summarizeProtocolTree === true;
-                break;
-            }
+                return this.submitMessage(
+                    type,
+                    // back-compat: ADO #1385: pass content as-is in future.
+                    typeof contents === "string" ? contents : JSON.stringify(contents),
```

Comment:
can we do the same thing for signals? It will keep the layers consistent across our message types

Reply:
slightly unrelated, but we also process signals in the delta manger, specifically we json parse them, and loose data, like the ref seq, which prevent us for leveraging ref seq on signals in the runtime. maybe this should be a parallel work stream to get the treatment between the two symmetrical, and ideally basically transparent to the loader layer.

Reply:
I agree, but do not want to make these PRs more complex. 
https://dev.azure.com/fluidframework/internal/_workitems/edit/1841